docs/application/native/guides/text-input/voice-control.md

```diff
@@ -661,8 +663,8 @@ To create a command list and commands:
 
    - You can use the `vc_cmd_list_get_current()` function to get the current command in an output parameter.
 
-   	**Note**  
-   When you get the command handle with the `vc_cmd_list_get_current()` function, do not release it. To release the command handle, call the `vc_cmd_list_remove()` function before the `vc_cmd_destroy()` function.
+   	> **Note**  
```

Comment:
The indentation on line 178 and 179 should be same. FYI - This note is not broekn on your repository page.

Reply:
Done